pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -786,6 +786,9 @@ public PartitionedTopicMetadata getPartitionedMetadata(
             @ApiParam(value = "Is check configuration required to automatically create topic")
             @QueryParam("checkAllowAutoCreation") @DefaultValue("false") boolean checkAllowAutoCreation) {
         validateTopicName(tenant, namespace, encodedTopic);
+        if (authoritative && checkAllowAutoCreation) {
```

Comment:
I don't think we should include `authoritive` here? https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/LookupOptions.java#L28-L31
https://github.com/apache/pulsar/blame/master/site2/website/versioned_docs/version-2.8.1/developing-binary-protocol.md#L478-L480

Reply:
sorry, I got it wrong. I changed it back.